owncloudApp/src/main/java/com/owncloud/android/presentation/adapters/filelist/FileListAdapter.kt

```diff
@@ -373,8 +379,8 @@ class FileListAdapter(
     }
 
     interface FileListAdapterListener {
-        fun onItemClick(ocFile: OCFile, position: Int)
-        fun onLongItemClick(ocFile: OCFile, position: Int): Boolean = true
+        fun onItemClick(ocFileWithSyncInfo: OCFileWithSyncInfo, position: Int)
```

Comment:
We only need the info about the file, but leaving it like this allows us to get rid of the `OCFile` dependency in this adapter, which I think it's beneficial. Also, it makes everything in this class more consistent since we use always the same entity.